MAUI/Scheduler/migration.md

```diff
@@ -975,6 +975,12 @@ this.Content = scheduler;
 <td>{{'[ResourceIds]()'| markdownify }}</td>
 <td>Gets or sets the resource ids property for the schedule appointment to allocate the corresponding resource to the appointment.</td></tr>
```

Comment:
Update in this table for TextColor and check and update any other places also 
https://help.syncfusion.com/maui/scheduler/appointments#scheduler-appointment-source-mapping

Reply:
Updated TextColor